### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by kianmeng**

### CODE REVIEW

File: src/zope/interface/tests/test_interface.py [hunk 1]
```diff
@@ -563,7 +563,7 @@ class IBar(Interface):
 
     def test_multiple_inheritance_no_interfaces(self):
         # If we extend an object that implements interfaces,
-        # plus ane that doesn't, we do not interject `Interface`
+        # plus and that doesn't, we do not interject `Interface`
```
Comment (jugmac00) [inline src/zope/interface/tests/test_interface.py:566]: That change makes no sense. Should be `one` I guess.

**Review by jugmac00 — DISMISSED** [review]

File: src/zope/interface/_zope_interface_coptimizations.c [hunk 2]
```diff
@@ -776,7 +776,7 @@ typedef struct {
 
 static struct PyMethodDef ib_methods[] = {
   {"__adapt__", (PyCFunction)__adapt__, METH_O,
-   "Adapt an object to the reciever"},
+   "Adapt an object to the receiver"},
```
Comment (jugmac00) [inline src/zope/interface/_zope_interface_coptimizations.c:779]: @jamadden or @d-maurer - I am not sure whether this change could have an undwanted side-effect. Could any of you please give a final review on this PR? Thansk!

**Comment by d-maurer** [comment]
Jürgen Gmach wrote at 2022-1-4 02:02 -0800:
***@***.*** commented on this pull request.
>
>
>
>> @@ -776,7 +776,7 @@ typedef struct {
>
> static struct PyMethodDef ib_methods[] = {
>   {"__adapt__", (PyCFunction)__adapt__, METH_O,
>-   "Adapt an object to the reciever"},
>+   "Adapt an object to the receiver"},
>
***@***.*** or @d-maurer - I am not sure whether this change could have an undwanted side-effect.

The changed text will become the "docstring".
This change should not cause important side effects.

**Review by d-maurer — APPROVED** [review]

**Comment by icemac** [comment]
Thank you for this PR and for reviewing it. 😃

**Comment by kianmeng** [comment]
:tada: :tada: :tada: :tada: :tada:

**Pull request merged**